### PR TITLE
SEA-178: Swap glazing and flooring questions

### PR DESCRIPTION
**Pull request by laurasaunders1**
**Changes**
Floor insulation is now part of the construction type question (wall/ loft/ floor insulation) and glazing is it's own question as this is a slighter more logical ordering.

We have reached out to BRE to see if we can send them the flooring info as the main questionnaire does not currently use it (although boiler questionnaire does).

**Testing**
The tests all successfully run and the boiler and home question questionnaires display/ work as expected.

**Risks**
Fairly low risk, possible some data has not been removed from db.

**Docs**
Not needed.

### CODE REVIEW

File: user-site-server/src/main/resources/db/changelogs/2021-08-02-add-distinct-glazing-type-question.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                VALUES (1, '', 'Glazing type', '', 'glazing_type', '','','', 0, 'question');
+
+                SET @post_id =  LAST_INSERT_ID();
+
+                INSERT INTO wp_postmeta (post_id, meta_key, meta_value)
+                VALUES
+                    (@post_id, '_autoOpenQuestionReason', 'field_5a0c5ddf40b11'),
+                    (@post_id, 'autoOpenQuestionReason', '0'),
+                    (@post_id, '_questionReason', 'field_5a0179b00e182'),
+                    (@post_id, 'questionReason', 'Eligibility for certain grants is determined by your benefits status'),
```
Comment (TheMattBarnfield) [inline user-site-server/src/main/resources/db/changelogs/2021-08-02-add-distinct-glazing-type-question.xml:23]: is this correct?

**Review by TheMattBarnfield — APPROVED** [review]
Looks good! Thanks Laura :)

File: user-site-server/src/main/resources/db/changelogs/2021-08-02-add-distinct-glazing-type-question.xml (hunk 1 above)
Comment (laurasaunders1) [reply user-site-server/src/main/resources/db/changelogs/2021-08-02-add-distinct-glazing-type-question.xml:23]: Well that's a wonderful spot isn't it! Needs updating, I will change to "The type of windows that you have make a large difference to your energy efficiency" as this follows the current construction wording. 

I will also update the construction question text as it references windows.

Cheers!

Comment (laurasaunders1) [reply user-site-server/src/main/resources/db/changelogs/2021-08-02-add-distinct-glazing-type-question.xml]: I have updated the text used for the glazing question. I am going to manually update the construction question as it's very quick to do in the admin site. Once on staging I will check with Alec if he is happy with the updates or wants any additional changes.

**Review by TheMattBarnfield — APPROVED** [review]

**Pull request merged**